rfcs/0093-rfc-categories.md

```diff
@@ -0,0 +1,152 @@
+---
+feature: rfc_categories
+start-date: 2021-05-19
+author: David Arnold
+co-authors: (find a buddy later to help out with the RFC)
+shepherd-team: @kevnicox, @Mic92, @gytis-ivaskevicius
+shepherd-leader: @kevincox
+related-issues:
+category: process
+---
+
+# Summary
+[summary]: #summary
+
+The RFC process is amended with means to categorize RFCs into one of: _feature_, 
+_information_ or _process_, where each category sets different accents that
+improve the overall outcome of the process.
+
+# Motivation
+[motivation]: #motivation
+
+## Some issues are not addressed by the community
+
+There is no appropriate venue for it. When reading the current RFC process, it becomes
+clear that it was made primarily with purely technical decisions in mind. It might only
+be a perception, but this perception systemically skews what's being discussed and what
+not.
+
+As a result, important topics might not be adressed and settled by the community at large
+in a generally accepted procedure.
+
+Specifically, it is hard to propose a well coordinated experiment across the community,
+document and acknowledge design issues, record proof-generated insight, but also to
+amend the RFC process itself, the forum rules, the code of conduct or propose any other 
+binding changes to community workflows or infrastructure.
+
+**For example**, the [flake RFC](https://github.com/NixOS/rfcs/pull/49), which by many is
+considered a failed RFC process, might have benefited from a framework to transition into
+a general experiment in the form of an _informational_ RFC as soon as it had become
+clear that it won't be accepted as a _feature_ RFC. Its subsequent closure has rendered
+the entire flake experiment to a largely undocumented, unstructured, and intransparent
+process that still elicites strong opinions within the community. Clarity over RFC options
+and venues _might_ have helped mitigate this situation.
```

Comment:
Can you explain better how having categories would've helped in the case of flakes?

Reply:
Categories act like a conceptual map, as a traditional map, they help navigate the process. I can't tell if it _wpuld_ have helped, but navigating with a (better) map seems indeed capable of improving the outcome (of navigation).